src/Functions/FunctionsHashing.h

```diff
@@ -274,6 +274,54 @@ struct SipHash64Impl
     static constexpr bool use_int_hash_for_pods = false;
 };
 
+struct SipHash64KeyedImpl
+{
+    static constexpr auto name = "sipHash64Keyed";
+    using ReturnType = UInt64;
+    struct KeyType
```

Comment:
Minor cosmetic comment: Name `*Type` (or `*T`) is usually for generic or templated types, in this case, just `Key` would do the job.